build.js

```
@@ -0,0 +1,77 @@
+  const requireMatches = rawContent.match(/require\(\s*['"]\.\/\w+['"]\s*\)/g) || [];
+  const requireCount = requireMatches.length;
+  return { file, rawContent, exportKeys, requireCount };
+});
+
+// Sort modules so dependencies (with fewer requires) come first
+modules.sort((a, b) => a.requireCount - b.requireCount);
+
+// Generate src/index.js from src/core components
+const indexPath = path.join(__dirname, 'src', 'index.js');
+const componentNames = files.map(f => path.basename(f, '.js'));
+let indexLines = [];
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'indexLines' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)